sacn/messages/universe_discovery.py

```diff
@@ -27,6 +27,8 @@ def sourceName(self) -> str:
 
     @sourceName.setter
     def sourceName(self, sourceName: int):
```

Comment:
Not an issue with your code, but could you also change the type of `sourceName` from `int` to `str`?